subscriptions/spec.md

```diff
@@ -860,14 +863,22 @@ For example:
 }
 ```
 
-###### `sql` filter dialect
+###### 3.2.4.1.2 Optional Filter Dialects
+
+The support of the following dialect is OPTIONAL for implementations of
```

Comment:
s/dialect/dialects/ just to be consistent with the plural-ness of the section header